ontology/uco/observable/observable.ttl

```diff
@@ -380,14 +380,56 @@ observable:ApplicationFacet
 			sh:nodeKind sh:Literal ;
 			sh:path observable:applicationIdentifier ;
 		] ,
+		[
+			sh:datatype observable:ApplicationVersion ;
+			sh:nodeKind sh:BlankNodeOrIRI ;
+			sh:path observable:installedVersion ;
+		]
+		;
+	sh:targetClass observable:ApplicationFacet ;
+	.
+
+observable:ApplicationVersion
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf observable:ObservableObject ;
+	rdfs:label "ApplicationVersion"@en ;
+	rdfs:comment "An application version represents a specific version of an application."@en ;
+	sh:targetClass observable:ApplicationVersion ;
+	.
+
+observable:ApplicationVersionFacet
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf core:Facet ;
+	rdfs:label "ApplicationVersionFacet"@en ;
+	rdfs:comment "An application version facet is a grouping of characteristics unique to a particular software program version."@en ;
+	sh:property
+		[
+			sh:datatype xsd:dateTime ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
+			sh:path observable:installDate ;
+		] ,
+		[
+			sh:datatype xsd:dateTime ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
+			sh:path observable:uninstallDate ;
+		] ,
 		[
 			sh:datatype xsd:string ;
 			sh:maxCount "1"^^xsd:integer ;
+			sh:minCount "1"^^xsd:integer ;
```

Comment:
We have [another proposal open](https://github.com/ucoProject/UCO/issues/428) about some required properties being a potentially harmful user experience.  I will be removing this `minCount` property.

Cc: @eoghanscasey